lib/use-cases/create-customer-url.js

```diff
@@ -0,0 +1,33 @@
+import { Token } from 'lib/domain';
+import { nanoid } from 'nanoid';
+
+export default class CreateCustomerUrl {
+  constructor({ planGateway }) {
+    this.planGateway = planGateway;
+  }
+
+  async execute({ planId }) {
+    const existingPlan = await this.planGateway.get({ id: planId });
+
+    const currentToken = existingPlan.customerTokens.find(
+      token => token.shared === false
+    );
+
+    if (currentToken) {
+      return {
+        customerPlanUrl: `${process.env.NEXT_PUBLIC_URL}/c/plans/${existingPlan.id}?token=${currentToken.token}`
```

Comment:
This is repeated again on line 30, maybe we could do something like this?

```
  const planBaseUrl = `${process.env.NEXT_PUBLIC_URL}/c/plans/${existingPlan.id}?token=`;
  ...
  return {
    customerPlanUrl: `${planBaseUrl}${currentToken.token}`
  }
```

Reply:
👌